### DISCUSSION TITLE
client branch is shifted to frontend branch

### DISCUSSION THREAD

**Discussion by Neo9050** [AlphacodelabsDemo/Task_Manager#24] [Q&A]
All the developement in  [client branch](https://github.com/AlphacodelabsDemo/Task_Manager/tree/client) is shifted to [frontend branch](https://github.com/AlphacodelabsDemo/Task_Manager/tree/frontend)... so from now on work on frontend branch .

**Comment by kanhaiyaverma2022** [comment]
I am going to work on frontend 📝
